install_travis_dependencies.sh

```diff
@@ -16,6 +16,7 @@ sudo apt-get install -y --force-yes gradle
 if [[ $languages == *"python"* ]]; then
   sudo apt-get install -y --force-yes python-dev python-pip
   sudo pip install --upgrade pip wheel virtualenv virtualenvwrapper --quiet
+  sudo pip install -r requirements.txt
```

Comment:
Actually, the virtual env wrapper is used by the test runner, what I'm doing here is installing dependencies globally and using it before the test runner starts with its job (which depends on the virtual env).
Also, I don't think I should depend on a virtual env as I'm linting files `5.x.py` and `6.x.py` (meaning all `.py` files), there's no need to create an env for something globally.